src/api/v1beta1/properties.go

```diff
@@ -90,11 +90,41 @@ func (dk *DynaKube) IsActiveGateMode(mode CapabilityDisplayName) bool {
 	return false
 }
 
-func (dk *DynaKube) KubernetesMonitoringMode() bool {
+func (dk *DynaKube) ActiveGateServiceAccountOwner() string {
+	if dk.IsKubernetesMonitoringCapabilityEnabled() {
+		return string(KubeMonCapability.DeepCopy().DisplayName)
+	} else {
+		return "activegate"
+	}
+}
+
+func (dk *DynaKube) ActiveGateServiceAccountName() string {
+	return "dynatrace-" + dk.ActiveGateServiceAccountOwner()
+}
+
+func (dk *DynaKube) IsKubernetesMonitoringCapabilityEnabled() bool {
 	return dk.IsActiveGateMode(KubeMonCapability.DisplayName) || dk.Spec.KubernetesMonitoring.Enabled
 }
 
-func (dk *DynaKube) NeedsStatsd() bool {
+func (dk *DynaKube) IsRoutingCapabilityEnabled() bool {
```

Comment:
Could the names not have the workd `Capability`? E.g., `IsRoutingEnabled`? Including `Capability` everywhere seems a bit redundant

Reply:
That was my first draft, but because its on the Dynakube I wanted to make it explicit that `Routing` is an ActiveGate thing, because `Routing` doesn't really mean anything if you don't know the context.

Thought about calling it `IsRoutingActiveGateEnabled`, maybe that is better, because `Capability` also means nothing without the context?

Reply:
`IsRoutingActiveGateEnabled` sounds good